website/source/docs/other/environment-variables.html.md

```diff
@@ -42,9 +42,10 @@ each can be found below:
     new versions of Packer. If you want to disable this for security or privacy
     reasons, you can set this environment variable to `1`.
 
--   `TMPDIR` (Unix) / `TMP` (Windows) - The location of the directory used for
-    temporary files (defaults to `/tmp` on Linux/Unix and
-    `%USERPROFILE%\AppData\Local\Temp` on Windows Vista and above). It might be
-    necessary to customize it when working with large files since `/tmp` is a
-    memory-backed filesystem in some Linux distributions in which case
-    `/var/tmp` might be preferred.
+-   `PACKER_TMP_DIR` - The directory used for temporary files during marshalling.
+    If unset, appends 'packer' to environment variables TEMP, TMP, or (windows:
+    LOCALAPPDATA) before falling back to the value of `configDir()/tmp` which
+    resolves to (unix: `$HOME/.packer.d`, windows: `%APPDAT%/packer.d`).
```

Comment:
This explanation is a bit inaccurate and references the source code (`configDir()`).

Reply:
updated and I think clearer in intent.